README.md

```diff
@@ -77,32 +71,36 @@ export REGISTRY_USERNAME=<USER>
 export REGISTRY_PASSWORD=<PASSWORD>
 ```
 
+You also need to fill the github token to retrieve the Hashicorp plugin
+```bash
+export GITHUB_TOKEN=<TOKEN>
+```
+
 ## 2.2. Namespaces
 
-Set environment variables to specify what namespace Orchesrate, its dependencies, and tools will be deployed. Note: all the releases could be deployed in the same namespace. Example:
+Set environment variables to specify what namespace Orchesrate and its dependencies will be deployed. Note: all the releases could be deployed in the same namespace. Example:
 
 ```bash
 export ORCHESTRATE_NAMESPACE=orchestrate-demo
-export VAULT_OPERATOR_NAMESPACE=hashicorp-vault
-export VAULT_NAMESPACE=hashicorp-vault
 ```
 
-Optionally, specifiy the namespace where the Prometheus and grafana stack will be deployed
+Optionally, specifiy the namespace where Vault Operator, Vault, Prometheus and Grafana stack will be deployed
 ```
+export VAULT_OPERATOR_NAMESPACE=vault
```

Comment:
is there any advantage in using two separate namespaces? if not, I would rather merge both namespaces into a single one for simplicity

Reply:
Yes it will be better. I just removed it